MdeModulePkg/Library/UefiBootManagerLib/BmLoadOption.c

```diff
@@ -240,6 +246,27 @@ structure.
   UnicodeSPrint (OptionName, sizeof (OptionName), L"%s%04x", mBmLoadOptionName[Option->OptionType], Option->OptionNumber);
 
   VariableAttributes = EFI_VARIABLE_BOOTSERVICE_ACCESS | EFI_VARIABLE_RUNTIME_ACCESS | EFI_VARIABLE_NON_VOLATILE;
+  if (Option->OptionType == LoadOptionTypePlatformRecovery) {
+    VariableAttributes = EFI_VARIABLE_BOOTSERVICE_ACCESS | EFI_VARIABLE_RUNTIME_ACCESS;
+  }
+
+  Status = gRT->SetVariable (
+                  OptionName,
+                  &gEfiGlobalVariableGuid,
+                  VariableAttributes,
+                  VariableSize,
+                  Variable
+                  );
+  FreePool (Variable);
+
+  // MU_CHANGE START: Don't register a variable policy if the SetVariable call failed
+  if (EFI_ERROR (Status)) {
+    return Status;
+  }
+
+  // MU_CHANGE END
+
+  // MU_CHANGE - Move RegisterBasicVariablePolicy after SetVariable for OptionName

```

Comment:
Did this change get removed intentionally? Looks like `RegisterBasicVariablePolicy ` is still before SetVariable call.

Reply:
I think this commit is saying the above SetVariable(..) call - not the lower one.